README.md

```diff
@@ -154,31 +183,31 @@ We'll load a local csv into CKAN DataStore instance.
 
 We'll assume you have:
 
-* a local CKAN setup and running at https://localhost:5000
+* a local CKAN setup and running at http://localhost:5000
 * datastore enabled. If you are using Docker, you might need to expose your Postgres Instance port. For example, add the following in your `docker-composer.yml` file:
+
 ```yml
 db:
   ports:
       - "5432:5432"
 ```
-(Useful to know: it is possible to access the Postgres on your Docker container. Run `docker ps` and you should see a container named `docker-ckan_db`, which corresponds to CKAN database. Run `docker exec -it CONTAINER_ID bash` and then `psql -U ckan` to access the corresponding Postgres instance).
 
-* Now you need to set up some information on Airflow. Access your local Airflow Connections panel on `http://localhost:8080/admin/connection/`. Create a new connection named `ckan_postgres` with your datastore information. For example, assuming your `CKAN_DATASTORE_WRITE_URL=postgresql://ckan:ckan@db/datastore`, use the following schema:
+(Useful to know: it is possible to access the Postgres on your Docker container. Run `docker ps` and you should see a container named `docker-ckan_db`, which corresponds to the CKAN database. Run `docker exec -it CONTAINER_ID bash` and then `psql -U ckan` to access the corresponding Postgres instance).
+
+* Now you need to set up some information on Airflow. Access your local Airflow Connections panel at http://localhost:8080/admin/connection/. Create a new connection named `ckan_postgres` with your datastore information. For example, assuming your `CKAN_DATASTORE_WRITE_URL=postgresql://ckan:ckan@db/datastore`, use the following schema:
 
 ![Connection configuration](docs/resources/images/aircan_connection.png)
 
 * We also need to set up two environment variables for Airflow. Access the Airflow Variable panel and set up `CKAN_SITE_URL` and your `CKAN_SYSADMIN_API_KEY`:
 
 ![Variables configuration](docs/resources/images/aircan_variables.png)
 
-
 [TODO PARAMETERIZE VARS]
 [TODO PARAMETERIZE PATHS]
 
+Then, create a dataset called `aircan-example` using this script:
```

Comment:
I didn't make changes to that example as I think it wasn't complete, right? I couldn't follow along with this one.